auto_nag/scripts/component.py

```diff
@@ -43,15 +43,26 @@ def sort_columns(self):
         return lambda p: (-p[3], -int(p[0]))
 
     def get_bz_params(self, date):
-        params = {
-            'component': 'Untriaged',
+        start_date, end_date = self.get_dates(date)
+
+        return {
             # Ignore bugs for which somebody has ever modified the product or the component.
             'n1': 1, 'f1': 'product', 'o1': 'changedafter', 'v1': '1970-01-01',
             'n2': 1, 'f2': 'component', 'o2': 'changedafter', 'v2': '1970-01-01',
+
+            # Ignore closed bugs.
             'bug_status': '__open__',
-        }
 
-        return params
+            # Get recent General bugs, and all Untriaged bugs.
+            'j3': 'OR',
+            'f3': 'OP',
+                'f4': 'OP',
```

Comment:
did you check that AND is the default operator for f4 ?

Reply:
Yep, I think AND is always the default.
I'll put 'j4' just to make this more readable.

Reply:
I think you're right (about AND is the default) but just in case...